index_old.html

```diff
@@ -78,6 +78,8 @@
                         <i class="fa fa-external-link" style="color: #ffffff;"></i> SDKs
                     </a>
                     <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                        <a class="dropdown-item"
```

Comment:
I know that this page is not being used, but, since it is in the repository, I have updated it anyway.